### ISSUE TITLE
Crash when create database

### ISSUE THREAD

**Issue by niuhf0452** [zendesk/maxwell#697]
Maxwell crashes when I create an new database.

Steps to reproduce this error:

* Prepare an empty mysql.
* Setup maxwell which watches on database 'A'.
* Run mysql, maxwell, zookeeper and kafka.
* Execute sql on mysql like this: `CREATE DATABASE A`

Then maxwell crash.

Logs:

```
java.lang.NullPointerException: null
 	at java.util.regex.Matcher.getTextLength(Matcher.java:1283) ~[?:1.8.0_131]
 	at java.util.regex.Matcher.reset(Matcher.java:309) ~[?:1.8.0_131]
 	at java.util.regex.Matcher.<init>(Matcher.java:229) ~[?:1.8.0_131]
 	at java.util.regex.Pattern.matcher(Pattern.java:1093) ~[?:1.8.0_131]
 	at com.zendesk.maxwell.MaxwellFilter.filterListsInclude(MaxwellFilter.java:111) ~[maxwell-1.10.4.jar:1.10.4]
 	at com.zendesk.maxwell.MaxwellFilter.matchesTable(MaxwellFilter.java:123) ~[maxwell-1.10.4.jar:1.10.4]
 	at com.zendesk.maxwell.MaxwellFilter.matches(MaxwellFilter.java:127) ~[maxwell-1.10.4.jar:1.10.4]
 	at com.zendesk.maxwell.MaxwellFilter.matches(MaxwellFilter.java:149) ~[maxwell-1.10.4.jar:1.10.4]
 	at com.zendesk.maxwell.schema.ddl.ResolvedSchemaChange.shouldOutput(ResolvedSchemaChange.java:26) ~[maxwell-1.10.4.jar:1.10.4]
 	at com.zendesk.maxwell.replication.AbstractReplicator.processQueryEvent(AbstractReplicator.java:106) ~[maxwell-1.10.4.jar:1.10.4]
 	at com.zendesk.maxwell.replication.MaxwellReplicator.processQueryEvent(MaxwellReplicator.java:367) ~[maxwell-1.10.4.jar:1.10.4]
 	at com.zendesk.maxwell.replication.MaxwellReplicator.getRow(MaxwellReplicator.java:341) ~[maxwell-1.10.4.jar:1.10.4]
 	at com.zendesk.maxwell.replication.AbstractReplicator.work(AbstractReplicator.java:157) ~[maxwell-1.10.4.jar:1.10.4]
 	at com.zendesk.maxwell.util.RunLoopProcess.runLoop(RunLoopProcess.java:27) ~[maxwell-1.10.4.jar:1.10.4]
 	at com.zendesk.maxwell.Maxwell.startInner(Maxwell.java:188) ~[maxwell-1.10.4.jar:1.10.4]
 	at com.zendesk.maxwell.Maxwell.start(Maxwell.java:136) ~[maxwell-1.10.4.jar:1.10.4]
 	at com.zendesk.maxwell.Maxwell.main(Maxwell.java:209) ~[maxwell-1.10.4.jar:1.10.4]
```

**Comment by ericwush** [comment]
@niuhf0452 I couldn't reproduce this error. Maybe include your config file? From the stack trace it is likely the table name is null for some reason. Does it ring any bell?

**Comment by ericwush** [comment]
Close the issue, fixed in #720
